apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -122,9 +122,9 @@ mandatory.claims.warning.msg.1=You need to provide consent for
 mandatory.claims.warning.msg.2=all the mandatory claims
 mandatory.claims.warning.msg.3=in order to proceed
 privacy.policy.cookies=Cookie Policy
-privacy.policy.cookies.short.description=After a successful sign in, we use a cookie in your browser to track your session. You can refer our
+privacy.policy.cookies.short.description=We use browser cookies to track your session to give better experience. You can refer our
 privacy.policy.general=Privacy Policy
-privacy.policy.privacy.short.description=By signing in, you agree to our
+privacy.policy.privacy.short.description=By signing in, you are agree to our
```

Comment:
This has to be `agreeing` right?

Reply:
Will revert it to `you agree` instead. As it's the standard